config_test.cc

```diff
@@ -1,40 +1,40 @@
 #include "gtest/gtest.h"
+#include <string.h>
 #include "config.h"
 
 TEST(ServerConfigTest, SimpleConfig) {
   ServerConfig config("test_config_1");
   EXPECT_EQ(config.GetPortNo(), 12345);
+  std::cout << config.ToString() << "     " << "Server is running on Port: 12345";
+  EXPECT_EQ(0, config.ToString().compare(std::string{"Server is running on Port: 12345"}));
 }
 
 TEST(ServerConfigTest, PortTooLargeConfig) {
   try{
   	ServerConfig config("test_config_2");
-  	EXPECT_EQ(config.GetPortNo(), 80);
   }
   catch (PortRangeException &e)
   {
-	std::cout << e.what();
+	EXPECT_EQ(0, strcmp(e.what(), "Port number out of range."));
   }
 }
 
 TEST(ServerConfigTest, NoPortConfig) {
   try{
   	ServerConfig config("test_config_3");
-  	EXPECT_EQ(config.GetPortNo(), 80);
   }
   catch (InvalidConfigException &e)
   {
-	std::cout << e.what();
+	EXPECT_EQ(0, strcmp(e.what(), "Port number missing in config file."));
   }
 }
 
 TEST(ServerConfigTest, EmptyConfig) {
   try{
   	ServerConfig config("test_config_4");
-  	EXPECT_EQ(config.GetPortNo(), 80);
   }
   catch (InvalidConfigException &e)
   {
-	std::cout << e.what();
+	EXPECT_EQ(0, strcmp(e.what(), "Port number missing in config file."));
   }
```

Comment:
I have thought about fixtures but I think it's not really worth the extra code for this small class.